forge/db/views/DeviceGroup.js

```diff
@@ -0,0 +1,63 @@
+module.exports = function (app) {
+    app.addSchema({
+        $id: 'DeviceGroupSummary',
+        type: 'object',
+        properties: {
+            id: { type: 'string' },
+            name: { type: 'string' },
+            description: { type: 'string' },
+            model: { type: 'string' }
+        }
+    })
+    function deviceGroupSummary (group) {
+        // if (Object.hasOwn(group, 'get')) {
+        //     group = group.get({ plain: true })
+        // }
+        if (group.toJSON) {
+            group = group.toJSON()
+        }
+        const result = {
+            id: group.hashid,
+            name: group.name,
+            description: group.description,
+            deviceCount: group.deviceCount || 0
+        }
+        return result
+    }
+
+    app.addSchema({
+        $id: 'DeviceGroup',
+        type: 'object',
+        allOf: [{ $ref: 'DeviceGroupSummary' }],
+        properties: {
+            createdAt: { type: 'string' },
+            updatedAt: { type: 'string' },
+            DeviceGroup: { type: 'object', additionalProperties: true }
+        },
+        additionalProperties: true
+    })
+    function deviceGroup (group) {
+        if (group) {
+            let item = group
+            if (item.toJSON) {
+                item = item.toJSON()
+            }
+            const filtered = {
+                id: item.hashid,
+                name: item.name,
+                description: item.description,
+                Application: item.Application ? app.db.views.Application.applicationSummary(item.Application) : null,
```

Comment:
We don't have capitalized properties anywhere else on the API. This should be `application`.